server/system/aws.js

```diff
@@ -49,7 +51,39 @@ const updateEmail = (oldEmail, newEmail) => {
   return _sendAwsRequest(_getAwsClient().adminUpdateUserAttributes(params))
 }
 
+const sendEmail = async (from, to, subject, body) => {
+  // Create sendEmail params 
+  const params = {
+    Destination: {
+      ToAddresses: [to]
+    },
+    Message: {
+      Body: {
+        Html: {
+          Charset: 'UTF-8',
+          Data: body
+        },
+      },
+      Subject: {
+        Charset: 'UTF-8',
+        Data: subject
+      }
+    },
+    Source: from,
+    ReplyToAddresses: [from],
+  }
+
+  // Create the promise and SES service object
+  const sendEmailPromise = new aws.SES({
+    apiVersion: '2010-12-01',
+    region: process.env.COGNITO_REGION,
+  }).sendEmail(params).promise()
+
+  return sendEmailPromise
```

Comment:
sendEmailPromise is a redundant variable

Reply:
Removed the variable